src/asphalt/core/cli.py

```diff
@@ -67,6 +80,37 @@ def run(configfile, unsafe: bool, loop: Optional[str], service: Optional[str]) -
         ), "the document root element must be a dictionary"
         config = merge_config(config, config_data)
 
+    # Read the configuration from the CLI
+    for cli_conf in set_:
+        if "=" not in cli_conf:
+            click.echo(f"Configuration must be set with '=', got: {cli_conf}")
+            raise click.Abort()
+
+        key, value = cli_conf.split("=", 1)
+        ks = key.split(".")
```

Comment:
I think we could use a regex to split the section identifiers while respecting the `\.`.

Reply:
I think I cannot do better than https://github.com/asphalt-framework/asphalt/pull/50/commits/30bda25731a2a7130c4170ba4f57cfa91c1edf2b, my regex knowledge is rather limited :smile: